sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveCatalogs.scala

```diff
@@ -160,6 +170,11 @@ class ResolveCatalogs(val catalogManager: CatalogManager)
         writeOptions = c.options.filterKeys(_ != "path"),
         orCreate = c.orCreate)
 
+    case insert @ InsertIntoStatement(
+        UnresolvedRelation(nameParts @ NonSessionCatalog(catalog, tableName)), _, _, _, _) =>
```

Comment:
shall we check temp view here? INSERT INTO works for temp views.

Reply:
`NonSessionCatalog` will convert temp views to v2 here, just like the other plans.